dataprofiler/tests/profilers/test_order_column_profile.py

```diff
@@ -413,3 +415,40 @@ def test_json_encode_after_update(self):
         )
 
         self.assertEqual(serialized, expected)
+
+    def test_json_decode(self):
+        fake_profile_name = None
+        expected_profile = OrderColumn(fake_profile_name)
+
+        serialized = json.dumps(expected_profile, cls=ProfileEncoder)
+        deserialized = load_column_profile(json.loads(serialized))
+
+        utils.assert_profiles_equal(deserialized, expected_profile)
+
+    def test_json_decode_after_update(self):
+        fake_profile_name = "Fake profile name"
+
+        # Build expected orderColumn
+        df_order = pd.Series(["za", "z", "c", "a"])
```

Comment:
might have be interesting to have this ordered and then the update below keep it ordered and validating the output expectation order. Maybe can quick add.

Reply:
I see what hes saying there `order` attribute is not consistent after updating